resources/views/receipt.blade.php

```diff
@@ -41,7 +41,7 @@
 
         .table td {
             padding: 8px;
-            line-height: 20px;
+            line-height: 14px;
```

Comment:
I decreased the line-height here slightly to account for the extra items on the invoice.